src/main/java/kg/jarkyn/cobspecserver/StreamHandler.java

```diff
@@ -0,0 +1,20 @@
+package kg.jarkyn.cobspecserver;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class StreamHandler {
+    public static void write(OutputStream outputStream, byte[] message) {
+        try {
+            outputStream.write(message);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } finally {
+            try {
+                outputStream.close();
```

Comment:
very good question, i can't remember why i put it there
will have to look at it agin and adjust